cli/src/command/maintenance.rs

```diff
@@ -70,6 +95,11 @@ impl ExonumCommand for Maintenance {
             Action::ClearCache => {
                 Action::clear_cache(self.node_config.clone(), self.db_path.clone())?
             }
+            Action::RestartMigration { ref service_name } => Action::restart_migration(
+                self.node_config.clone(),
+                self.db_path.clone(),
```

Comment:
You can get rid of these clones if you'll change signature of the `restart_migration` function.